index.html

```diff
@@ -10,7 +10,7 @@
         group: "vc",
 
         // specification status (e.g., WD, NOTE, etc.). If in doubt use ED.
-        specStatus: "FPWD",
+        specStatus: "ED", // really, withdrawn, see:  https://www.w3.org/2021/Process-20211102/#discontinuedREC
```

Comment:
There is no such thing as a separate spec status for discontinued REC. Just leave this field as "ED", the Echidna process will turn this into a Working draft, and it will stay that way. And that is perfectly fine.

Reply:
merged your suggestion, resolving.